packages/runtime/datastore/src/test/channelStorageService.spec.ts

```diff
@@ -68,15 +69,15 @@ describe("ChannelStorageService", () => {
                 },
             },
         };
-        const storage: Pick<IDocumentStorageService, "read"> = {
-            read: async (id: string) => {
-                return id;
+        const storage: Pick<IDocumentStorageService, "readBlob"> = {
+            readBlob: async (id: string) => {
+                assert.fail();
             },
         };
         const ss = new ChannelStorageService(tree, storage);
 
         assert.equal(await ss.contains("nested/foo"), true);
         assert.deepStrictEqual(await ss.list("nested/"), ["foo"]);
-        assert.equal(await ss.read("nested/foo"), "bar");
+        assert.equal(blobToString(await ss.readBlob("nested/foo")), "bar");
```

Comment:
Naming: I feel like "blob" has too meaning:
1. binary blob, aka array
2. IBlob

Should this be arrayToString? or stringFromArray?
Array seems like better represent what we work with.
readBlob() name might be confusing, but I'd guess once everything settles, we could rename it to plain "read" and thus names would be consistent:

stringFromArray(stroage.read(...)

What do you think?

Reply:
sure, I'm thinking bufferToString, since my other helper function is called toBuffer